concepticondata/concepticon.tsv

```diff
@@ -1930,7 +1930,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1929	STUDY	Cognition	To follow a course of study; to be enrolled at an institute of learning.	Action/Process	
 1930	RAVINE	The physical world	A deep narrow valley or gorge in the earth's surface worn by running water.	Person/Thing	
 1931	BEESWAX	Animals	a yellowish or dark brown wax secreted by honeybees for constructing honeycombs.	Person/Thing	
-1932	GUESS	Cognition	To succeed to know, or discover, by way of surmises.	Action/Process	
+1932	GUESS	Cognition	To make a surmise.	Action/Process	
```

Comment:
Maybe just "to suppose"? Make a surmise seems a bit too technical.

Reply:
One definition we have is close to mine, the other is basically "to estimate".
I guess I'll just include it all, i.e. "To suppose, to estimate or to make a surmise."